ci/requirements-3.6_CONDA_BUILD_TEST.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+source activate pandas
+
+echo "install 36 CONDA_BUILD_TEST"
+
+conda install conda-build
+
+conda build ./conda.recipe/ --numpy 1.11 --python 3.6 -q
```

Comment:
rather than the actual build line here, but this in install_travis_script; call this BUILD_TEST_CONDA (rather than BUILD_TEST) so you can disambiguate, rename the other to BUILD_TEST_PIP.